doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,269 @@
 
 //##############################################################################
 
+/**
+        @page TOPP_example_qualitycontrol Qualitycontrol
+
+        @section TOPP_example_qualitycontrol_section Qualitycontrol
+
+        @b Goal: Check the quality of the data.
+
+        The QualityControl TOPP tool is able to check the quality of different aspects if enough data is provided.
+        Depending on the given data this tool executes different metrics that have access to all of their required
+        files (see section 'Metrics'). They will then add new metavalues to the data and write all possiblbe
+        information into the mzTab output file.
+
+        @section TOPP_example_qualitycontrol_section2 Workflow
+
+        @image html TOPP_qualitycontrol.png
+        @image latex TOPP_qualitycontrol.png "" width=10cm
+
+
+        @section  TOPP_example_qualitycontrol_section3 Metrics
+        This Table shows what each of the included metrics does, what they need to be executed and what they add to
+        the data or what they store in their output.
+        <TABLE border=1>
+
+        <TR>
+            <TH colspan="2">
+                <b><br>Contaminants</b>
+                </TH>
+        </TR>
+        <TR>
+            <TD>
+                <b>Essential input data:</b> Fasta file (Contaminants database), PostFDR FeatureXML
```

Comment:
can you explain what a `PostFDR FeatureXML` is? *i.e. a featureXML file after FDR filtering.
A good place to do that is above the table  - just briefly explain all input types